modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
     - node.type.branch
   module:
     - address
+    - content_moderation
```

Comment:
Why content moderation added? there is no request to add it in the original issue.